README.md

```diff
@@ -80,28 +87,33 @@ images.get_image("185-34")
 
 ```
 
-## Osm data provider
+### Osm data provider
+
 Provides osm data for a defined coordinate area, or for the coordinates of a geoTiff image
 
-### Usage in console
+#### Usage in console
+
 ```bash
-py OsmDataProvider -h
+py osmdataprovider -h
 ```
 
-### Example in console
+#### Example in console
+
 ```bash
-py OsmDataProvider ../data/in/osm/vector -p ../data/in/ortho/wgs84/DOP25_LV95_2240-33_2015_1_15.tif
+py osmdataprovider ../data/in/osm/vector -p ../data/in/ortho/wgs84/DOP25_LV95_2240-33_2015_1_15.tif
 ```
 
-### Example in Python
+#### Example in Python
 
 You can use this module in python as single executor, the same way as you use it in the console
+
 ```python
 import OsmDataProvider
```

Comment:
this will not work, because you need to import the whole package => small letters
```suggestion
import osmdataprovider
osmdataprovider.Provider(target="../data/in/osm/vector", imagepath="../data/in/ortho/wgs84/DOP25_LV95_2240-33_2015_1_15.tif")

```